modules/manually-create-identity-access-management.adoc

```diff
@@ -260,15 +207,28 @@ apiVersion: v1
 kind: Secret
 metadata:
   namespace: openshift-image-registry
-  name: installer-cloud-credentials
+ifdef::aws[]
```

Comment:
This is a general question for all clouds, in step 6.
The are the following in CR sample.
name: <component-secret>
    namespace: <component-namespace>.

But there are the following in secret sample(to take aws as example). 
namespace: openshift-image-registry
  name: aws-creds

Is it intended?